Bottom: de5d8312acdef47fb1376540aeceb8b1c94ef25c
Top:    e2aa888f6686bc6a5877ebed7eada17e85b8ea16
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-07 14:11:38 -0500

Refresh of use-docker-named-networks

---
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index a745a5ef..aa571feb 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -110,6 +110,7 @@
       - name: '{{ docker_network_frontend }}'
     ports: '{{ exim4_port_args }}'
     image: '{{ exim4_image_name }}:{{ docker_image_tag }}'
+    purge_networks: true
     restart_policy: '{{ docker_restart_policy }}'
     state: started
     volumes:
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index 06f0a979..16b79804 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -132,6 +132,7 @@
     image: '{{ redis_image_name }}:{{ docker_image_tag }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+    purge_networks: true
 
 - name: start container (postgres)
   docker_container:
@@ -142,6 +143,7 @@
     image: '{{ gitlab_db_image_name }}:{{ docker_image_tag }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+    purge_networks: true
 
 - name: start container (gitlab)
   docker_container:
@@ -152,6 +154,7 @@
     ports: '{{ gitlab_port_args }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+    purge_networks: true
     volumes-from:
       - '{{ openssl_dv_name }}'
       - '{{ gitlab_dv_name }}'
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 8f9bbbc2..1a36c616 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -142,6 +142,7 @@
       MYSQL_PASSWORD: '{{ mantisbt_db_password }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+    purge_networks: true
 
 - name: wait for initialization to complete
   shell: >
@@ -168,6 +169,7 @@
     ports: '{{ mantisbt_port_args }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+    purge_networks: true
     env:
       MANTISBT_HOSTNAME: '{{ mantisbt_hostname }}'
       MANTISBT_MAIL_USER: 'mantis'
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 14ba4153..0b7d5895 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -113,6 +113,7 @@
     networks:
       - name: '{{ docker_network_frontend }}'
     state: started
+    purge_networks: true
 
 - name: wait for mysql initialization to complete
   shell: >
@@ -139,6 +140,7 @@
     ports: '{{ wiki_port_args }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+    purge_networks: true
     volumes:
       - '{{ openssl_dv_name }}:/etc/ssl/private:z'
       - '{{ wiki_dv_name }}/var/www/html:z'
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index edb5e80a..fb60a09a 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -122,6 +122,7 @@
     image: '{{ openldap_image_name }}:{{ docker_image_tag }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+    purge_networks: true
 
 - name: start container (phpldapadmin)
   docker_container:
@@ -138,6 +139,7 @@
       LDAP_HOSTNAME: '{{ openldap_hostname }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+    purge_networks: true
     image: '{{ phpldapadmin_image_name }}:{{ docker_image_tag }}'
 
 # *****************************************************************************
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index d45e1dc9..324b0baa 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -84,6 +84,7 @@
     ports: '{{ svn_port_args }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+    purge_networks: true
     volumes:
       - '{{ openssl_dv_name }}:/etc/ssl/private:z'
       - '{{ svn_dv_name }}:/var/lib/svn:z'
